### DISCUSSION TITLE
All Components red, how to analyze?

### DISCUSSION THREAD

**Discussion by tkalmar** [vuejs/language-tools#928] [Q&A] [answered]
In my current nuxtjs project all Components in a vue file are displayed red wit type any
![image](https://user-images.githubusercontent.com/971164/152311194-c2ee0e67-5345-4c6d-bb3f-311fffb18d58.png)
How can i get typings and code completion to work?
Using volar 0.31.1 in takeover mode.

**Comment by johnsoncodehk** [comment, marked as answer]
Can you reproduce this problem in https://github.com/johnsoncodehk/volar-starter?

If you're using vue 2, please make sure follow setup: https://github.com/johnsoncodehk/volar/tree/master/extensions/vscode-vue-language-features#usage

**Comment by tkalmar** [comment]
Not reproducible with the starter :/ Rechecked the instructions and found out i missed the settings in tsconfig for vue compiler compatibility.
Thanks a lot
